runtimes/server-interface/lsp.ts

```diff
@@ -58,6 +58,17 @@ export type PartialServerCapabilities<T = any> = Pick<
     | 'signatureHelpProvider'
 >
 export type PartialInitializeResult<T = any> = {
+    /**
+     * Information about the server respresented by @type {Server}.
+     * serverInfo is used to differentiate servers internally in the system and is not exposed to the client.
+     */
+    serverInfo?: {
```

Comment:
Shall we use different name for this data? Since this `serverInfo` is different from regular LSP, it's purpose may be a bit confusing. Maybe some alternative like `serverSystemInfo` or `internalServerInfo` would be better?

Reply:
I can rename, but I do not like any of those options. LSP server is kind of a composite of feature servers, that's why I liked the idea of borrowing the property name from LSP server level. Also, given feature servers are not exposed to the client directly, their server info is also not exposed.